main/social/search.php

```diff
@@ -118,9 +118,22 @@
             } else {
                 $user_icon = Display::return_icon('teacher.png', get_lang('Teacher'), null, ICON_SIZE_TINY);
             }
-
+            
+            $plugin = BuyCoursesPlugin::create();
+            $includeServices = $plugin->get('include_services') === 'true';
+            $serviceNode = null;
+            $class = "";
+            $icon = "";
```

Comment:
This is micro-optimization and might not be useful, but from a pure algorithmic point of view, " triggers an interpretation of the string, whereas ' just prints it, so it takes less effort for the system to compute '' than "".